app/enterprise/0.32-x/edition-versioning.md

```diff
@@ -20,4 +20,4 @@ Changes to the released version of Kong Enterprise Edition occur as follows:
 
 - **Z** is incremented against previously released versions of Kong Enterprise Edition when a critical fix must be implemented against an old release.
 
-The nature of this versioning scheme is such that the scope of semantic weight of changes in each release of Kong Enterprise Edition **cannot be interpreted solely through the change in version number**. Because of this, we strongly encourage Enterprise Edition users to consult the [Kong Enterprise Changelog](/enterprise/latest/changelog) and [Upgrading Kong Enterprise](https://support.konghq.com/hc/en-us/articles/115009693288-Upgrading-Kong) before upgrading.
+The nature of this versioning scheme is such that the scope of semantic weight of changes in each release of Kong Enterprise Edition **cannot be interpreted solely through the change in version number**. Because of this, we strongly encourage Enterprise Edition users to consult the [Kong Enterprise Changelog](/enterprise/changelog) and [Upgrading Kong Enterprise](https://support.konghq.com/hc/en-us/articles/115009693288-Upgrading-Kong) before upgrading.
```

Comment:
The link for Upgrading Kong Enterprise is broken here too.

Reply:
@rainest , does https://support.konghq.com/hc/en-us/articles/115009693288-Upgrading-Kong contain any content for customers with credentials?

Reply:
```suggestion
The nature of this versioning scheme is such that the scope of semantic weight of changes in each release of Kong Enterprise Edition **cannot be interpreted solely through the change in version number**. Because of this, we strongly encourage Enterprise Edition users to consult the [Kong Enterprise Changelog](/enterprise/changelog) and [Upgrading Kong Enterprise](/enterprise/0.32-x/upgrades-migrations/) before upgrading.
```